src/components/organisms/AssetActions/Consume.tsx

```diff
@@ -59,6 +61,7 @@ export default function Consume({
   const [hasDatatoken, setHasDatatoken] = useState(false)
   const [isConsumable, setIsConsumable] = useState(true)
   const [assetTimeout, setAssetTimeout] = useState('')
+  const [fileConnectivity, setFileConnectivity] = useState<boolean>()
```

Comment:
we are repeating this state setting and updating 3 times, suggesting it should be abstracted out for DRY reasons. Could be done inside a new `ConnectivityHelp` component under `atoms`